reference.md

```diff
@@ -44,6 +44,7 @@ Presence conditions applicable to fields and files:
 * REQUIRED - The file or the field MUST be included in the dataset, and a value MUST be provided in the field for each record.
 * OPTIONAL - The file or the field MAY be omitted from the dataset, and a value MAY be omitted in the field for any record.
 * Conditionally REQUIRED - The file or the field is REQUIRED under certain conditions, which are outlined in the file or field description. Outside of these conditions, the file or the field is OPTIONAL.
+* Conditionally OPTIONAL - The file or the field is OPTIONAL under certain conditions, which are outlined in the file or field description. Outside of these conditions, the file or the field is forbidden.
```

Comment:
I'm not sure we need `Conditionally OPTIONAL` vs just OPTIONAL. I understand the difference but it might just be confusing